timelock-server/src/testCommon/java/com/palantir/atlasdb/timelock/NamespacedClients.java

```diff
@@ -59,7 +61,13 @@ static NamespacedClients from(String namespace, ProxyFactory proxyFactory) {
     default TimelockService timelockService() {
         return RemoteTimelockServiceAdapter.create(
                 namespacedTimelockRpcClient(),
-                namespacedConjureTimelockService());
+                namespacedConjureTimelockService(),
+                lockWatchEventCache());
+    }
+
+    @Value.Default
+    default LockWatchEventCache lockWatchEventCache() {
```

Comment:
it's unclear to me why you need this here? is this used anywhere? or are you only creating it here to be used inside `timelockService` above?

Reply:
For tests that don't exist yet I guess